Makefile

```diff
@@ -52,17 +52,13 @@ check: verify test-unit test-integration
 #
 # Example:
 #   make verify
-verify: verify-fmt verify-govet verify-imports verify-gometalinter verify-sec
+verify: verify-fmt verify-imports verify-sec
```

Comment:
why are we dropping govet and gometalinter?  do we not feel they add value? are they causing problems?

Reply:
gometalinter is deprecated and being replaced with golangci-lint (which throws a ton of linting errors when run) and will take some time to fix all of them so that it will pass.

golangci-lint includes a staticcheck that is "go vet on steroids" so i figured we would use that instead.